tests/integration/components/sl-button-test.js

```diff
@@ -0,0 +1,206 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
+
+moduleForComponent( 'sl-button', 'Integration | Component | sl button', {
+    integration: true
+});
+
+const mockStreamService = {
+    send() {}
+};
+
+const template = hbs`
+    {{#sl-button}}
+        Default Text
+    {{/sl-button}}
+`;
+
+/**
+ * While it appears that core Ember functionality is being tested this test is
+ * ensuring that the implied contract about which DOM element is rendered is
+ * adhered to.
+ */
+
```

Comment:
Remove this blank line to more clearly associate the comment above with the specific test below.